stratum/lib/security/credentials_manager.cc

```diff
@@ -9,68 +9,121 @@
 #include <utility>
 
 #include "absl/memory/memory.h"
+#include "absl/time/clock.h"
+#include "absl/time/time.h"
 #include "gflags/gflags.h"
 #include "stratum/glue/logging.h"
 #include "stratum/lib/macros.h"
 #include "stratum/lib/utils.h"
 
-DEFINE_string(ca_cert_file, "", "CA certificate path");
-DEFINE_string(server_key_file, "", "gRPC Server private key path");
-DEFINE_string(server_cert_file, "", "gRPC Server certificate path");
+DEFINE_string(server_ca_cert_file, "", "Path to CA certificate file");
```

Comment:
There probably isn't much of a need for a separate `server_ca_cert_file` and `client_ca_cert_file`.

I guess the only use case would be a program that is both a server and client where the CA's are different, but that seems a bit contrived to me.

Reply:
Reverted back to single cert